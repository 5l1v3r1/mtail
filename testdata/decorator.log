Dec
Jan
Dec
Apr
Dec
Oct
