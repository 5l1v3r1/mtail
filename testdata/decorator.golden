counter a 3 2018-06-10T00:32:42Z
counter b 3 2018-06-10T00:32:42Z
counter c 3 2018-06-10T00:32:42Z
